README.md

```diff
@@ -11,6 +11,7 @@ loaded in yt.  It is written to provide both scripting and interactive access.
 
 * Free software: BSD license
 * Documentation: https://yt-idv.readthedocs.io.
+* Source code: https://github.com/yt-project/yt_idv
```

Comment:
i added this since the README.md gets included as the readme of the readthedocs page. I like having an obvious link to the repo on the readthedocs homepage :)

Reply:
great idea